opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1325,6 +1325,15 @@ Defaulted grid coordinates is not allowed for COMPDAT as part of ACTIONX)"
         return this->snapshots[timeStep].groups.get(groupName);
     }
 
+    bool Schedule::hasChoke(const std::string& groupName, std::size_t timeStep) const {
+        if (this->snapshots[timeStep].network.get().has_node(groupName)) {
+            return this->snapshots[timeStep].network.get().node(groupName).as_choke();
+        }
+        else {
+            return 0;
+        }
```

Comment:
You could probably shorten this to
```C++
const auto& network = this->snapshots[timeStep].network();
return network.has_node(nodeName) && network.node(nodeName).as_choke();
```
if you'd like.  This does get into the realms of "eye of the beholder" for readers.

Reply:
Your proposed code looks more neat to me so I followed that also the renaming of the parameter name